static/js/topic_page/map_tile.tsx

```diff
@@ -274,8 +283,9 @@ function draw(
     }
     return place.name + ": " + value;
   };
+  document.getElementById(MAP_ID).innerHTML = "";
```

Comment:
This should not be required. Maybe this was added after the map Id's were hardcoded?